src/main/java/net/snowflake/client/core/HttpUtil.java

```diff
@@ -114,16 +117,26 @@ public class HttpUtil {
 
   @SnowflakeJdbcInternalApi
   public static Duration getConnectionTimeout() {
-    return Duration.ofMillis(
-        SystemUtil.convertSystemPropertyToIntValue(
-            JDBC_CONNECTION_TIMEOUT_IN_MS_PROPERTY, DEFAULT_HTTP_CLIENT_CONNECTION_TIMEOUT_IN_MS));
```

Comment:
Do we still need `JDBC_CONNECTION_TIMEOUT_IN_MS_PROPERTY` and `JDBC_SOCKET_TIMEOUT_IN_MS_PROPERTY` constants?